student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(age, name, classroom)
+    super(age, name)
+    @classroom = classroom
+  end
```

Comment:
Kindly include the `parent_permission` parameter in your `initialize` and `super` constructors.

- Kindly make sure that the default value for `name` is  "Unknown" and the default value for `parent_permission`  is set to "true".